### DISCUSSION TITLE
LibSQL vs better-sqlite3

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#760] [Q&A] [answered]
Hi,

What's the difference between [LibSQL](https://github.com/tursodatabase/libsql) and `better-sqlite3`? Is it worth using LibSQL instead?

**Comment by kentcdodds** [comment, marked as answer]
As far as I understand it, LibSQL is a fork of sqlite, better-sqlite3 uses sqlite. Feel free to make an example of using LibSQL if you like! We only use better-sqlite3 for the cache anyway so there probably aren't many differences.

**Comment by thefrana** [comment]
We have done internal research in our company and it turns out from our experiments that LibSQL does not yet has broad ecosystem. While they are compatible with SQLite, there are not many tools that can actually use the LibSQL features only.

The most notable feature of LibSQL in my eyes is ability to modify schema fully. However, we have not found an ORM that actually supports this feature and we remained on SQLite3 for now.
